src/mqt/benchviewer/backend.py

```diff
@@ -183,35 +188,69 @@ def filter_database(self, benchmark_config: BenchmarkConfiguration) -> list[str]
             | (self.database["benchmark"].isin(selected_nonscalable_benchmarks))
         ]
 
-        if benchmark_config.indep_qiskit_compiler:
-            db_tmp1 = db_tmp.loc[(db_tmp["indep_flag"]) & (db_tmp["compiler"] == "qiskit")]
+        if benchmark_config.indep_tket_compiler:
```

Comment:
Here is the wrong compiler flag checked, must be `indep_bqskit_compiler` instead of the tket one.

Reply:
Hi @nquetschlich, noted. I'll make the corrections.

Reply:
@nquetschlich I've changed the order of compilers and optimization level to make BQSKit last as below and fixed the bug on the incorrect bqskit flag check.

<img width="1328" alt="Screenshot 2024-01-24 at 11 12 07 AM" src="https://github.com/cda-tum/mqt-bench/assets/5851344/befafdc0-f1c3-4dad-9294-bf21825cc21e">